src/scss/Main.scss

```diff
@@ -3,6 +3,9 @@
 @import "Start";
 @import "City";
 @import "Header";
+@import "Map";
+@import "Tooltip";
+@import "Detail";
```

Comment:
why those files are in `Main.scss` :suspect: 
Why not just import them as `import './Map.scss` in react component file?